test/check-sosreport

```diff
@@ -35,11 +34,7 @@ class TestSOS(MachineCase):
         b.click('[data-target="#sos"]')
         b.wait_visible("#sos")
 
-        # HACK - https://bugzilla.redhat.com/show_bug.cgi?id=1278807
```

Comment:
Is this fixed in f23? the test timed out and there has been no final answer in the bug tracker so far.